### ISSUE TITLE
Yazi does not boot in Powershell

### ISSUE THREAD

**Issue by irvingoujAtDevolution** [sxyazi/yazi#844]
### What system are you running Yazi on?

Windows

### What terminal are you running Yazi in?

Powershell 7, Powershell 5

### Did you try the latest main branch to see if the problem got fixed?

Tried, but the problem still

### `yazi --debug` output

<details>

```sh
PS C:\Users\jou> yazi --debug
←[>q←_Gi=31,s=1,v=1,a=q,t=d,f=24;AAAA←\←[c
```

</details>

### Describe the bug

nothing to report, it does not even run.

### Expected Behavior

at least it should be runing.

### To Reproduce

```sh
PS C:\Users\jou> yazi --debug
←[>q←_Gi=31,s=1,v=1,a=q,t=d,f=24;AAAA←\←[c
```

**Comment by sxyazi** [comment]
Hi, thanks for the report!

I'm not quite sure I understand what you mean, could you record a gif and let me see what was happening?

**Comment by sxyazi** [comment]
Hi, I made a PR trying to fix this, https://github.com/sxyazi/yazi/pull/845, let me know if it works for you!

**Comment by irvingoujAtDevolution** [comment]
https://github.com/sxyazi/yazi/assets/139169536/da089032-dfc9-4ee2-b56a-64a9fbe07bde

Still not working

**Comment by sxyazi** [comment]
You need to use the code from that PR, `cargo --git` installs the latest main branch, not the code in that PR.

**Comment by irvingoujAtDevolution** [comment]
sorry, I thought you merged it, let me try again

**Comment by irvingoujAtDevolution** [comment]
yes, it worked! Thanks!

**Comment by sxyazi** [comment]
Thanks for the test! Let me merge it then
